documentation/infrastructure_crl.markdown

```diff
@@ -0,0 +1,15 @@
+---
+layout: default
+title: "Puppet Server: Intermediate CA"
+canonical: "/puppetserver/latest/infrastructure_crl.html"
+---
+
+# Infrastructure certificate revocation list
+
+The Puppet Server CA can create a smaller CRL that contains only revocations of those nodes that agents are expected to talk to during normal operations, for example, compile masters or hosts that agents connect to as part of agent-side functions. Puppet Server CA can distribute that CRL to agents, rather than the CRL it maintains with all node revocations.
```

Comment:
I don't love "smaller".  I guess "leaner" would be better, but then it sounds like some kind of meat :(

Reply:
"trimmed-down"?

Reply:
it also makes sense sans adjective